### PR TITLE
Add SoundCommand Class & Remove Extensions from Require

### PR DESCRIPTION
**Pull request by DanielRX**
Made the sound commands simpler and more generic (They can support mp3 files btw)
Removed the extension from the require calls as it's actually more robust: [Example](https://willi.am/blog/2014/10/12/understanding-nodes-require-function/)

### CODE REVIEW

**Comment by samhza** [comment]
Why was this closed?

**Comment by DanielRX** [comment]
I closed it as it wasn't ready to be merged, there were some issues I had missed

**Comment by DanielRX** [comment]
The sound player fix: If a sound file is less than a multiple of 1 second (2.5seconds) the bot closes the connection and leaves the voice channel before the sound is complete, meaning the full audio file is not run.

File: commands/soundboard/remotesound.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const MusicCommand = require("../../classes/musicCommand");
+const soundPlayer = require("../../utils/soundplayer");
+
+class RemoteSoundCommand extends MusicCommand {
```
Comment (TheEssem) [inline commands/soundboard/remotesound.js:4]: This command seems neat, but could pose a security risk and as such should be restricted to the bot owner only.

**Comment by TheEssem** [comment]
> The sound player fix: If a sound file is less than a multiple of 1 second (2.5seconds) the bot closes the connection and leaves the voice channel before the sound is complete, meaning the full audio file is not run.

A bit of a tip for the future: you can convert an existing PR into a draft to have it remain open but also make it not able to be merged. :)

File: commands/soundboard/remotesound.js (hunk 1 above)
Comment (DanielRX) [reply commands/soundboard/remotesound.js:4]: Yeah the goal was to enable something like "custom sounds" by allowing the server owner (in a self host) to add new sound commands directly within discord

**Comment by TheEssem** [comment]
Closing due to inactivity and the fact that the codebase has switched over to ECMAScript modules.

**Pull request closed without merging**